src/GoTrueClient.ts

```diff
@@ -558,6 +562,27 @@ export default class GoTrueClient {
     }
   }
 
+  private async _handleOpenIDConnectSignIn(
+    { IDToken, nonce, clientID, issuer, provider }: OpenIDConnectCredentials
+  ): Promise<{
+    session: Session | null
+    user: User | null
+    error: ApiError | null
+  }> {
+    if (IDToken && nonce && ( (clientID && issuer) || provider) ) {
```

Comment:
```suggestion
    if (id_token && nonce && ( (client_id && issuer) || provider) ) {
```
For consistency in naming.